curvefs/proto/mds.proto

```diff
@@ -169,6 +172,36 @@ message RefreshSessionResponse {
     repeated topology.PartitionTxId latestTxIdList = 2;
 }
 
+message DLockValue {
+    required string owner = 1;
+    required uint64 expireTime = 2;
+}
+
+message GetLatestTxIdRequest {
+    optional bool lock = 1;
+    optional uint32 fsId = 2;
+    optional string fsName = 3;
+    optional string uuid = 4;
```

Comment:
why all the fields is optional ?

Reply:
> why all the fields is optional ?

there are two ways to invoke `CommitTX` RPC, one is lock, one is commit directly and the another one is commit with lock.